frontend/test/__runner__/generate-cypress-snapshots.js

```diff
@@ -0,0 +1,37 @@
+const cypress = require("cypress");
+
+const getConfig = baseUrl => {
+  return {
+    configFile: "frontend/test/__support__/e2e/cypress-snapshots.json",
+    config: {
+      baseUrl,
+    },
+  };
+};
+
+const generateSnapshots = async (baseUrl, exitFunction) => {
+  const snapshotConfig = getConfig(baseUrl);
+
+  try {
+    const results = await cypress.run(snapshotConfig);
```

Comment:
Destructuring could make this more readable: `{ status, message, failures, totalFailed }`.

Reply:
True.
But I wanted to wait until I finish with the actual Cypress runner module. We might want to extract and re-use this logic there as well.